src/config.h

```diff
@@ -335,6 +338,25 @@
 // Delay transmission for an N second offset, counting from the scheduled time set with TIME_SYNC_SECONDS.
 #define HORUS_V2_TIME_SYNC_OFFSET_SECONDS 0
 
+/**
+ * CATS mode settings
+ */
+#define CATS_CALLSIGN CALLSIGN
+#define CATS_SSID 29 // 0 - 255
```

Comment:
Please add comments with links to the CATS documentation that specified SSIDs and icons. It'll help users a lot! :)